cdap-docs/admin-manual/source/installation/index.rst

```diff
@@ -30,16 +30,15 @@ Installation and configuration instructions for either **specific distributions*
 
 - :ref:`Installation for Amazon EMR <admin-installation-emr>`
 
-  - **Advanced Topics:** enabling perimeter security, CDAP high availability, CDAP
-    Explore engines, and enabling Kerberos is currently not available for Amazon EMR
+  - **Advanced Topics** (enabling perimeter security, CDAP high availability, CDAP
+    Explore engines, and enabling Kerberos) are currently not available for Amazon EMR
 
 ..
 
 - :ref:`Installation using Apache Ambari <admin-installation-ambari>`
 
   - :ref:`Advanced Topics: <ambari-installation-advanced-topics>`
-    enabling Kerberos; enabling perimeter security, CDAP high availability, and CDAP Explore engines
-    are currently not available for Ambari
+    enabling perimeter security, Kerberos, CDAP high availability, and CDAP Explore engines
```

Comment:
Does that mean that the text at the link (http://docs.cask.co/cdap/current/en/admin-manual/installation/ambari.html#hive-execution-engines) needs to be changed?